### PR TITLE
[KAN-16]:About Page Third Section

### PR DESCRIPTION
**Pull request by poulami-saha**
Section implemented:
image

### CODE REVIEW

**Review by sehajwadhwa — DISMISSED** [review]

**Comment by joanwolk** [comment]
The display at 960px and above gives a functional min-width of 992px to the row of brands, which means it cuts off some of the images on the edges of the page. It also overflows the expected gutter space at the edge of the page.

Screenshot 2024-07-01 at 12 59 12 PM

File: src/components/About/Clients.module.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+.container {
+  background: #012f34;
```
Comment (joanwolk) [inline src/components/About/Clients.module.css]: use CSS variables

File: src/components/pages/About/About.jsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
+import Clients from "../../About/Clients";
 // import styles from "./About.module.css";
 
 const About = () => {
-  return <div>About</div>;
+  return <Clients />;
```
Comment (joanwolk) [inline src/components/pages/About/About.jsx]: If you want to make adding the other components easy for whoever comes after you, wrap this in a [Fragment](https://react.dev/reference/react/Fragment):

```
return (
  <>
    <Clients />
  </>
);
```

**Review by joanwolk — CHANGES_REQUESTED** [review]

File: src/components/About/Clients.module.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    height: 150px;
+    width: 150px;
+    top: 0;
+    left: 0;
+    transform: translate(0, 0%);
+  }
+  .news {
+    flex-direction: row;
+    justify-content: space-between;
+  }
```
Comment (joanwolk) [inline src/components/About/Clients.module.css:114]: These properties are already inherited from the `@media (min-width: 768px)` rules for `.news` (lines 65-69). If you reduce the gap for tablet sizes (see my other comment), you should set `gap: 3rem;` here though.

File: src/components/About/Clients.module.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    height: 308px;
+    max-width: 100%;
+    overflow-x: hidden;
+  }
+  .title {
+    margin-bottom: 48px;
+  }
+  .news {
+    flex-direction: row;
+    gap: 3rem;
```
Comment (joanwolk) [inline src/components/About/Clients.module.css]: This gap could be reduced slightly to fix the missing/cramped gutter space from 768px-800px. Changing to 2.5rem looks good to me.

**Review by joanwolk — CHANGES_REQUESTED** [review]

File: src/components/About/Clients.module.css (hunk 3 above)
Comment (poulami-saha) [reply src/components/About/Clients.module.css:114]: I have kept it 2.5rem now as with initial 4rem it was overflowing, and as you said with 3rem , the tablet view is not correct. so 2.5rem for both tablet and desktop now looks good

**Review by joanwolk — APPROVED** [review]

**Pull request merged**